aten/src/ATen/templates/TypeDefault.cpp

```diff
@@ -23,6 +23,18 @@ namespace TypeDefault {
 
 }  // namespace TypeDefault
 
+TORCH_LIBRARY_IMPL(_, AutogradCPU, m) {
+  m.fallback(torch::CppFunction::makeFallthrough());
+}
+
+TORCH_LIBRARY_IMPL(_, AutogradCUDA, m) {
+  m.fallback(torch::CppFunction::makeFallthrough());
+}
+
+TORCH_LIBRARY_IMPL(_, AutogradOther, m) {
+  m.fallback(torch::CppFunction::makeFallthrough());
+}
```

Comment:
These probably don't belong in the TypeDefault.cpp template. My guess for where they should go, is they should go in the same place the historic Autograd fallthrough lived.